frontend/utilities/endpoints.ts

```diff
@@ -43,6 +43,9 @@ export default {
     `/${API_VERSION}/fleet/device/${token}/software/install/${uuid}/results`,
   DEVICE_SOFTWARE_UNINSTALL: (token: string, softwareTitleId: number) =>
     `/${API_VERSION}/fleet/device/${token}/software/uninstall/${softwareTitleId}`,
+  // Used to view uninstall script results
+  DEVICE_SOFTWARE_SCRIPT_RESULTS: (token: string, scriptExecutionId: string) =>
+    `/${API_VERSION}/fleet/device/${token}/software/script/${scriptExecutionId}/results`,
```

Comment:
@iansltx my best guess on endpoint

Reply:
Got it. I'll be tweaking this but the params will be the same so easy swap. Response format will match existing script execution results endpoint 'cuz might as well.